app/blueprints/assessments/templates/macros/filter_options.html

```diff
@@ -73,6 +74,43 @@
                     {% endif %}
                 </div>
 
+                <div>
```

Comment:
These look like very similar patterns, could maybe extra them to a macro and re-use some code? Not too opinionated on it though, as maybe they'll diverge in the future